### PR TITLE
fix(EditorialNewsletter): Replace call to action w/ prominent link to trial

### PR DESCRIPTION
**Pull request by patrickvenetz**
Bildschirmfoto 2020-05-14 um 15 42 27

### CODE REVIEW

**Comment by patrickvenetz** [comment]
Not yet settled on wording.

File: src/templates/EditorialNewsletter/email/Header.js [hunk 1]
```diff
@@ -8,33 +27,59 @@ export default ({ meta }) => {
...
-      >
-        <a
-          href={`https://www.republik.ch${path ? path : `/${slug}`}`}
-          title='Im Web lesen'
+    <>
+      <tr>
+        <td
+          align='center'
+          valign='top'
+          style={{ borderBottom: !isCovid19 && `1px solid ${colors.divider}` }}
```
Comment (tpreusse) [inline src/templates/EditorialNewsletter/email/Header.js:35]: shouldn't pass false to react style afaik
```suggestion
          style={{ borderBottom: !isCovid19 ? `1px solid ${colors.divider}` : undefined }}
```

Comment (patrickvenetz) [reply src/templates/EditorialNewsletter/email/Header.js:35]: Get it. Thanks.

**Pull request merged**